helm-charts/easegress/templates/PersistentVolume.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   persistentVolumeReclaimPolicy: Delete
   storageClassName: easegress-storage
   local:
-    path: /opt/easegress
+    path: {{ $.Values.cluster.volumeLocalPath }}
```

Comment:
suggest to remove the `$` sign and use `{{ .Values.cluster.volumeLocalPath }}` instead.

Reply:
In the range block,  `{{ .Values.xxx }}` doesn't work.  I change to define another varuable.

Reply:
Cool, lesson learned. Look good to me.